appengine/pond_storage/common.py

```diff
@@ -67,17 +67,33 @@ class LeaderboardEntry(ndb.Model):
   ranking = ndb.IntegerProperty(required=True)
   instability = ndb.FloatProperty(required=True)
   duck_key = ndb.KeyProperty(kind=Duck, indexed=False)
-  is_dummy = ndb.ComputedProperty(lambda self: self.duck_key is None)
+  has_duck = ndb.ComputedProperty(lambda self: self.duck_key is not None)
```

Comment:
This was changed so that I could query with equality (==) for entries that have a duck